raiden/api/rest.py

```diff
@@ -234,24 +253,13 @@
         SettlementLightResourceByTokenAndPartnerAddress
     ),
     ("/payments_light", PaymentLightResource),
-    ("/light_client_messages", LightClientMessageResource, "Message polling"),
     ("/payments_light/create", CreatePaymentLightResource, "create_payment"),
+    ("/payments_light/unlock/<hexaddress:token_address>", UnlockPaymentLightResource),
```

Comment:
I think just for consistency that all the resources that are similar, like `/payments_light/...` should be together to keep an order on the file so when we search for it we can find it quickly

Reply:
totally aggre, I'll move `register_onchain_secret` here